crates/mod-kafka/Cargo.toml

```diff
@@ -4,9 +4,12 @@ version = "0.1.0"
 edition = "2021"
 
 [dependencies]
-anyhow = {workspace=true}
+anyhow.workspace=true
 config = {path="../config"}
 duration-serde = {path="../duration-serde"}
+futures.workspace=true
 mlua = {workspace=true, features=["vendored", "lua54", "async", "send", "serialize"]}
-rdkafka = {workspace=true}
-serde = {workspace=true}
+rdkafka.workspace=true
+serde.workspace=true
+tokio.workspace=true
+tracing.workspace=true
```

Comment:
I think we can trim this out with the API change to return the errors instead of logging them

```suggestion
```

Reply:
good catch, I'll remove it